### PR TITLE
[HOTFIX] NEXVIRTMOS-510

### PR DESCRIPTION
**Pull request by sfonn**
NEXVIRTMOS-510: Fix setup_bsp.sh failing when install kernel from deb

### CODE REVIEW

**Review by yangliang-intel — APPROVED** [review]
Looks good

File: guest_setup/ubuntu/unattend_ubuntu/setup_bsp.sh [hunk 1]
```diff
@@ -65,11 +65,11 @@ LOGE="logger -s -t $LOGTAG"
 declare -F "check_non_symlink" >/dev/null || function check_non_symlink() {
     if [[ $# -eq 1 ]]; then
         if [[ -L "$1" ]]; then
-            echo "Error: $1 is a symlink."
+            $LOGE echo "Error: $1 is a symlink."
```
Comment (XiaoHeSG) [inline guest_setup/ubuntu/unattend_ubuntu/setup_bsp.sh]: I think need remove echo

Comment (sfonn) [reply guest_setup/ubuntu/unattend_ubuntu/setup_bsp.sh]: fixed

**Review by XiaoHeSG — APPROVED** [review]
looks good to me

**Pull request merged**